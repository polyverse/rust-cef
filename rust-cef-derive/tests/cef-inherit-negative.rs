
#[macro_use]
extern crate rust_cef_derive;

#[cef_inherit]
union OnUnion{}

#[cef_inherit()]
struct EmptyInherit{}

#[cef_inherit()]
struct EmptyInherit2{}

fn main() {

}