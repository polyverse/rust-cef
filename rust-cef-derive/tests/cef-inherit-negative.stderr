error: This attribute only applies to Structs or Enums.
 --> $DIR/cef-inherit-negative.rs:5:1
  |
5 | #[cef_inherit]
  | ^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: 'cef_inherit' attribute must have no parameters or arguments. It must stand by itself.
 --> $DIR/cef-inherit-negative.rs:8:1
  |
8 | #[cef_inherit(Hello)]
  | ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)
